www/js/survey.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+angular.module('emission.main.survey', [])
+
+.controller('SurveyCtrl', function($scope, SurveyLaunch) {
+  $scope.$on('$ionicView.afterEnter', function() {
+    var surveyUrl = 'https://berkeley.qualtrics.com/jfe/form/SV_afnJgB5brT0zCo5';
+    var uuidEelementId = 'QR~QID56';
+    SurveyLaunch.startSurvey(surveyUrl, uuidEelementId);
+  })
+
+  $scope.startSurvey = function () {
```

Comment:
not really my problem, but if you are already displaying the survey on `afterEnter`, then why do you need a button?